1_fetch/src/find_nwis_sites.R

```diff
@@ -0,0 +1,19 @@
+get_drb_sites <- function(hucs,pcodes_select,site_tp_select){
+  
+  # hucs is a character vector containing the minor HUCs (hydrologic unit codes) over which to search. A minor HUC is 8 digits in length. 
```

Comment:
Very helpful to have these explanations for the function parameters. I have very limited experience with R, but if it's not too much of a burden, I think it would be good to follow the roxygen documentation format. I've seen that format in other R scripts that our team has used and everyone following the same format makes it really easy to understand quickly. https://cran.r-project.org/web/packages/roxygen2/vignettes/roxygen2.html

Reply:
In the code edits within this PR I've added roxygen-style documentation to each of the functions. Thanks for this great suggestion that will help all of us read and interpret the code!